### PR TITLE
Feat: new DS core components

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- new DS core components

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/design-system-core/src/interfaces/avatar.interface.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export type AvatarSrc = { url?: string; fallbackUrl?: string };
```
Comment (quininez) [inline ui/design-system-core/src/interfaces/avatar.interface.ts]: its better if we keep the interface definitions in the file where they are used

File: ui/design-system-core/src/components/ModerationAppErrorCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (quininez) [inline ui/design-system-core/src/components/ModerationAppErrorCard/index.tsx]: we should keep more generic components in the design system core, so either make it something like ErrorCard and make it more generic or implement it directly in the app

File: ui/design-system-core/src/interfaces/basicCardBox.interface.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export interface IBasicCardBox {
```
Comment (quininez) [inline ui/design-system-core/src/interfaces/basicCardBox.interface.ts]: same thing for interfaces, keep them with the component

File: ui/design-system-core/src/interfaces/moderationAppErrorCard.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export interface IModerationAppErrorCardProps {
```
Comment (quininez) [inline ui/design-system-core/src/interfaces/moderationAppErrorCard.ts]: interface -> component

File: ui/design-system-core/src/interfaces/basicpopover.interface.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export type OverflowType = 'auto' | 'hidden' | 'scroll' | 'visible';
```
Comment (quininez) [inline ui/design-system-core/src/interfaces/basicpopover.interface.ts]: interface -> component

File: ui/design-system-core/src/interfaces/appAvatar.interface.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import { IntegrationTypes } from '@akashaorg/typings/ui';
```
Comment (quininez) [inline ui/design-system-core/src/interfaces/appAvatar.interface.ts]: interface -> component

**Review by quininez — CHANGES_REQUESTED** [review]

File: ui/design-system-core/src/interfaces/avatar.interface.ts (hunk 1 above)
Comment (josenriagu) [reply ui/design-system-core/src/interfaces/avatar.interface.ts]: Alright. I'll fix those

**Review by quininez — APPROVED** [review]

**Pull request merged**